src/main/Config.cpp

```diff
@@ -246,8 +247,10 @@ Config::Config() : NODE_SEED(SecretKey::random())
     PEER_FLOOD_READING_CAPACITY = 200;
     FLOW_CONTROL_SEND_MORE_BATCH_SIZE = 40;
 
-    PEER_FLOOD_READING_CAPACITY_BYTES = 300000;
-    FLOW_CONTROL_SEND_MORE_BATCH_SIZE_BYTES = 100000;
+    // If set to 0, calculate automatically (this will be done after application
+    // startup as we need to load soroban configs)
+    PEER_FLOOD_READING_CAPACITY_BYTES = 0;
```

Comment:
regarding comment in HerderImp.cpp L2000 condition;  only if calculating defaults. LGTM